tests/scripts/test_repo_files.py

```diff
@@ -159,11 +159,14 @@ def do_test_links(self, path):
         codes = list()
         threads = list()
         for m in matches:
+            m = m if not m.startswith("#") else path.name + m
```

Comment:
What is this change for?

Reply:
Added a comment. Basically it is for links that appear in the docs as `[my link](#some-header-in-the-same-document)`. In some cases, i.e. when accessing a commit directly, the filename needs to be in the url, `https://github.com/[...]/README.md#some-header-in-the-same-document` instead of `https://github.com/[...]/#some-header-in-the-same-document`